### ISSUE TITLE
External links don't work in localized HTML

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1044]
Test case:
- [Ubuntu Packaging Guide](https://code.launchpad.net/~ubuntu-packaging-guide-team/ubuntu-packaging-guide/trunk) current snapshot (you can download the tarball using a button on [this page](http://bazaar.launchpad.net/~ubuntu-packaging-guide-team/ubuntu-packaging-guide/trunk/revision/);
- Spanish translation;
- Sphinx 1.1.3 with some backports and [Takayuki Shimizukawa](https://bitbucket.org/shimizukawa)'s [pull request 86](https://bitbucket.org/birkenfeld/sphinx/pull-request/86/fixed-955-footnote-i18n-translation-cause/diff).

External links work with English version, but with Spanish version they are converted to non-working internal links, i.e.

```
<a class="reference internal" href="#python-policy">Python policy</a>
```

instead of

```
<a class="reference external" href="http://www.debian.org/doc/packaging-manuals/python-policy/">Python policy</a>
```

Syntax of link used here:

```
Our packaging follows Debian’s `Python policy`_.
<...>

.. _`Python policy`: http://www.debian.org/doc/packaging-manuals/python-policy/
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1044
- Originally reported by: Dmitry Shachnev
- Originally created at: 2012-11-26T13:45:23.864

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-27 04:06:12+00:00_

It problem appeares only [Takayuki Shimizukawa](https://bitbucket.org/shimizukawa)'s [pull request 86](https://bitbucket.org/birkenfeld/sphinx/pull-request/86/fixed-955-footnote-i18n-translation-cause/diff) version.

I'm going to fix it.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-27 08:39:35+00:00_

[Dmitry Shachnev](https://bitbucket.org/mitya57), I fixed repoted error.
Please re-check [pull request 86](https://bitbucket.org/birkenfeld/sphinx/pull-request/86/fixed-955-footnote-i18n-translation-cause/diff).

**Comment by shimizukawa** [comment]
_From Dmitry Shachnev on 2012-11-27 10:40:43+00:00_

[Takayuki Shimizukawa](https://bitbucket.org/shimizukawa): That bug was not specific for your branch, but thanks a lot anyway for fixing it! Now it works!

May I suggest some grammar fixes for your branch? See http://paste.ubuntu.com/1391314/.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-27 11:09:37+00:00_

Thanks [Dmitry Shachnev](https://bitbucket.org/mitya57) !  I updated [pull request 86](https://bitbucket.org/birkenfeld/sphinx/pull-request/86/fixed-955-footnote-i18n-translation-cause/diff) again!

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-28 07:05:16+00:00_

waiting to accept https://bitbucket.org/birkenfeld/sphinx/pull-request/86.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-12-07 08:24:48+00:00_

fix #1044 refs #955: fix external/inline links with localization

→ <<cset c347b50330a18a9328c62b30aba3892de4a755db>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-12-07 08:24:49+00:00_

fix some grammar refs #1044. thanks [Dmitry Shachnev](https://bitbucket.org/mitya57) !

→ <<cset fb770e9013d79d5c6170f0a4e07efd3403e75b37>>
